snowplow-demo-kotlin/src/main/java/com/snowplowanalytics/snowplowdemokotlin/Demo.kt

```diff
@@ -0,0 +1,464 @@
+/*
```

Comment:
Nit: [Based on this convention](https://kotlinlang.org/docs/coding-conventions.html#source-code-organization), it seems to me that pure Kotlin projects shouldn't have the deep folder structure like java has with the `main/java/com/snowplowanalytics/snowplowdemokotlin` folder hierarchy and should only define folders for the modules inside the app.

I was also considering whether we should do this for the snowplow package, but with the mixed Kotlin and Java files, it's probably better to keep them in the same folder structure.